fix: DIA-1435: [FE, cust-Comet] Some errors on Prompt task list are not displayed gracefully on UI

nick-skriabin opened the pull request:
This pull request makes a small enhancement to the tooltip styling by limiting its maximum width.

* [`web/libs/ui/src/lib/Tooltip/Tooltip.module.scss`](diffhunk://#diff-1689043134504edaad1d5eb358066805af994052d48d6273eeefa63580a5bca6R21): Added a `max-width` property with a value of 250px to ensure tooltips do not exceed this width, improving readability.

<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

nick-skriabin committed:
Add max-width to the tooltip

nass600 commented:
@nick-skriabin @ricardoantoniocm Do we foresee any special case where we would like to override this `max-width`?

If we do I would recommend adding the tailwind class and allow merging from the outside with `twMerge` otherwise looks good

ricardoantoniocm commented:
> @nick-skriabin @ricardoantoniocm Do we foresee any special case where we would like to override this `max-width`?
> 
> If we do I would recommend adding the tailwind class and allow merging from the outside with `twMerge` otherwise looks good

Thanks for raising this, Nacho. 

Yes, I foresee that. Let's not set a max width in stone by ensuring it can be overwritten if necessary. Also, this particular change could affect how specific tooltips display info across the app. Let's make sure this change doesn't negatively affect the contents of tooltips across the app.

nick-skriabin commented:
@ricardoantoniocm @nass600 the width is customizable via `style` or `className` properties, so i don't see the problem here

nick-skriabin committed:
Allow setting different width via `className`

MihajloHoma commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14830445565)

MihajloHoma committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14830445565

MihajloHoma committed:
Merge branch 'develop' into 'fb-dia-1435/error-display'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14830445565

nick-skriabin committed:
Update text wrapping

yyassi-heartex commented on web/libs/ui/src/lib/Tooltip/Tooltip.module.scss:
```diff
@@ -18,6 +18,9 @@
   border-radius: 3px;
+  white-space: normal;
+  overflow-wrap: break-word;
+  max-width: 250px;
```
the ticket seems to be prompt task list specific but the change will apply to all tooltips - is this intended? I see class name is being passed in to tooltip now which is great and needed but why a sweeping change to address a specific issue?

nick-skriabin replied:
while yes, i think the rest of the tooltips will benefit from this change. it's not that big of a change anyway and most tooltips fall under the 250px min width

MihajloHoma commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14853321127)

MihajloHoma committed:
Merge branch 'develop' into 'fb-dia-1435/error-display'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14853321127

MihajloHoma committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14853321127

nick-skriabin committed:
Fix long text display

nick-skriabin committed:
Add interactive example

AndrejOros commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14878760207)

AndrejOros committed:
Merge branch 'develop' into 'fb-dia-1435/error-display'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14878760207

AndrejOros committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14878760207

nick-skriabin committed:
Add button states

nick-skriabin committed:
Add button type to tooltip example

nick-skriabin committed:
Fix text wrapping

nick-skriabin committed:
Fix overflow
